{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\ntest_alluredemo1.py:19: Exception"}, "start": 1592731463154, "stop": 1592731463155, "uuid": "237300d3-51b3-4c0e-942e-485a8a4ea055", "historyId": "62e3d0efc8226834b589024da8762ee8", "testCaseId": "17acedbab6e0d57f20e780d7d6084ef4", "fullName": "test_alluredemo1#test_broken", "labels": [{"name": "suite", "value": "test_alluredemo1"}, {"name": "host", "value": "LAPTOP-CH6E9HR5"}, {"name": "thread", "value": "21388-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_alluredemo1"}]}